app/controllers/api/v1/people_actions.rb

```diff
@@ -76,12 +76,12 @@ def supported_relationships
     end
 
     def included_relationships
-      IncludeParamHandler.new(params).call || PersonSerializer::SUPPORTED_RELATIONSHIPS
+      include_params_handler.included_relationships || PersonSerializer::SUPPORTED_RELATIONSHIPS
     end
 
     def other_included_relationships
       # Custom included relationships to avoid the people actions relationships
-      @other_included_relationships ||= IncludeParamHandler.new(params).call || Api::PeopleController::OTHER_SUPPORTED_RELATIONSHIPS
+      @other_included_relationships ||= IncludeParamHandler.new(params).included_relationships || Api::PeopleController::OTHER_SUPPORTED_RELATIONSHIPS
```

Comment:
can we use `include_params_handler.included_relationships` instead of `IncludeParamHandler.new(params).included_relationships`?